weblate/utils/views.py

```diff
@@ -80,6 +80,24 @@ def dispatch(self, request, *args, **kwargs):
         return super().dispatch(request, *args, **kwargs)
 
 
+def get_sort_name(request):
+    """Gets sort name."""
+    sort_dict = {
+        "position": _("Position"),
+        "priority": _("Priority"),
+        "num_words": _("Word count"),
+        "context": _("Context"),
+        "priority,position": _("Position and priority"),
```

Comment:
Shouldn't it be `-priority,position` as elsewhere?

Reply:
No. This is a dictionary without the sign. So that it doesn't matter what is the toggle sort. I thought that makes sense even for future?